helm-charts/docsum/README.md

```diff
@@ -26,7 +26,9 @@ helm install docsum docsum --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} --se
 # helm install docsum docsum --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} --values docsum/rocm-values.yaml
 # To use AMD ROCm device with TGI
 # helm install docsum docsum --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} --values docsum/rocm-tgi-values.yaml
-
+# helm install docsum docsum --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} --values docsum/gaudi-tgi-values.yaml ..
```

Comment:
Extra line duplicating one from above?

Reply:
There's now another duplicate line, but as Lianhao commented also, I'll resolve this.